.vscode/cspell.json

```diff
@@ -152,6 +152,15 @@
         "VMware",
         "vSphere"
       ]
+    },
+	{
+      "filename": "**/sdk/desktopvirtualization/**/*.cs",
+      "words": [
+        "logix",
+        "msix",
+        "ssoadfs",
+        "unhide"
```

Comment:
Are these words abbreviations in the SDK? Maybe we need to add them to the `rename-rules` if they are

Reply:
except the 'unhide' others are all common names of some industry product / standard